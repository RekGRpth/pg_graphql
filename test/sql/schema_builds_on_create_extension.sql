drop extension pg_graphql;
create extension pg_graphql;

select graphql.resolve($${ heartbeat }$$) -> 'data' ->> 'heartbeat' like '2%';
