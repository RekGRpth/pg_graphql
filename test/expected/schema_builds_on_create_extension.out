drop extension pg_graphql;
create extension pg_graphql;
select graphql.resolve($${ heartbeat }$$) -> 'data' ->> 'heartbeat' like '2%';
 ?column? 
----------
 t
(1 row)

